pkg/kbld/cmd/resolve.go

```diff
@@ -27,6 +27,7 @@ type ResolveOptions struct {
 
 	FileFlags        FileFlags
 	RegistryFlags    RegistryFlags
+	AllowedToBuild   bool
```

Comment:
Out of curiosity, was it coin toss or did you intentionally go with the affirmative flag name (e.g. as opposed to for example, `DisallowBuilds`).

Reply:
the flag itself is called "build". internal name is AllowedToBuild, which probably has to make sense for the default value of true.